Graphviz/DOT.sublime-syntax

```diff
@@ -1,62 +1,390 @@
+          | {{smoothtype_attributes_types}}
+          | {{style_attribute_types}}
+          )
+          (\")
+        )
+      scope: meta.string.dot string.quoted.double.dot
+      captures:
+        1: punctuation.definition.string.begin.dot
+        2: support.constant.dot
+        3: punctuation.definition.string.end.dot
+
+    - match: |-
+        (?x:
+          (\")
+          ((\#)(?:\h{3}|\h{6}|\h{8}))
+          (\")
+        )
+      scope: meta.string.dot string.quoted.double.dot
+      captures:
+        1: punctuation.definition.string.begin.dot
+        2: support.constant.dot color.dot
+        3: punctuation.definition.generic.dot
+        4: punctuation.definition.string.end.dot
+
+  literal-double-quoted-strings:
+    - match: \"
       scope: punctuation.definition.string.begin.dot
-      push:
-        - meta_scope: string.quoted.double.dot
-        - match: '"'
-          scope: punctuation.definition.string.end.dot
-          pop: true
-        - match: \\.
-          scope: constant.character.escape.dot
-        - match: (\\)$\n?
-          captures:
-            1: punctuation.separator.continuation.line.dot
-        - match: '[{}|]'
-          scope: punctuation.separator.memory-block.dot
-    - include: braces
-    - include: brackets
-    - include: embedded-html
-    - include: comments
+      push: literal-double-quoted-string-body
 
+  literal-double-quoted-string-body:
+    - meta_include_prototype: false
+    - meta_scope: meta.string.dot string.quoted.double.dot
+
+    - match: \"
+      scope: punctuation.definition.string.end.dot
+      pop: 1
+    - match: '(\\){{eol}}'
+      captures:
+        1: punctuation.separator.continuation.line.dot
+
+    - include: illegal-newline
+    - include: literal-double-quoted-string-escapes
+
+    # TODO: https://graphviz.org/doc/info/shapes.html#record
+    - match: \{
+      scope: punctuation.section.block.begin.memory.dot
+    - match: \}
+      scope: punctuation.section.block.end.memory.dot
+    - match: \|
+      scope: punctuation.separator.memory.dot
+    - match: (\<)(\w+)(\>)
+      scope: meta.annotation.label.dot
+      captures:
+        1: punctuation.definition.annotation.begin.dot
+        2: entity.name.label.record-based-node.dot
+        3: punctuation.definition.annotation.end.dot
+
+  literal-double-quoted-string-escapes:
+    - match: (\\)\"
+      scope: constant.character.escape.dot
+      captures:
+        1: punctuation.definition.string.escape.dot
+
+    - include: literal-string-escapes
+
+  literal-string-escapes:
+    # https://graphviz.org/docs/attr-types/escString/
+
+    # \\ = backslash
+    # \n, \l, \r = alignment (center, left, right)
+    # \N, \H, \T = node name, head node name, tail node name
+    # \G = graph name
+    # \E = edge name
+    # \L = object's label attribute
+    - match: (\\)[\\lnrEGHLNT]
+      scope: constant.character.escape.dot
+      captures:
+        1: punctuation.definition.string.escape.dot
```

Comment:
To avoid repeating myself, please refer to https://github.com/sublimehq/Packages/pull/4176#discussion_r1969801500